quill-jdbc-zio/src/test/scala/io/getquill/mock/ZioMockSpec.scala

```diff
@@ -202,7 +204,7 @@ class ZioMockSpec extends AnyFreeSpec with MockitoSugar { // with AsyncMockitoSu
     // closing autocommit bracket
     order.verify(conn).setAutoCommit(true)
     // connection close bracket
-    order.verify(conn).close()
+    order.verify(conn, times(1)).close()
```

Comment:
We ensure that the `.close` is called only once: in the "recovery code" (See https://github.com/zio/zio-quill/pull/2928/files#r1366847401), and not twice: in the "recovery code" and in the "scopedBestEffort" release code